rero_ils/modules/item_types/api.py

```diff
@@ -125,9 +125,10 @@ def exist_name_and_organisation_pid(cls, name, organisation_pid):
     def get_number_of_items(self):
         """Get number of items."""
         from ..items.api import ItemsSearch
-        return ItemsSearch() \
-            .filter('term', item_type__pid=self.pid) \
-            .source().count()
+        return ItemsSearch().filter('bool', should=[
+            Q('match', item_type__pid=self.pid),
+            Q('match', temporary_item_type__pid=self.pid)
+        ]).count()
```

Comment:
I think it would be better to use `term` instead of `match`:

```suggestion
        return ItemsSearch().query('bool', should=[
            Q('term', item_type__pid=self.pid),
            Q('term', temporary_item_type__pid=self.pid)
        ]).count()
```

@jma do you have an opinion?

Reply:
I agree